scripts/pipeline/static-linter-scan.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+readonly usage="Usage: static-linter-scan.sh --git-token <GHE token> --static-linter-version <Linter Version>"
+readonly WORK_DIR="${PWD}/linter-tool"
+
+function install_linter() {
+
+    # Download JSON of all versions
+    curl -H "Authorization: token ${GIT_TOKEN}" -H "Accept: application/vnd.github.v3.raw" -s https://github.ibm.com/api/v3/repos/IBMPrivateCloud/content-verification/releases > "${WORK_DIR}/versions.json"
+    
+    # Using jq, parse list of versions to find download for the version required
+    # Linter version is hard-coded here, for now
+    LINTER_TOOL_URL=$(jq -r '.[] | .assets | .[] |  select (.browser_download_url | endswith("v3.0.1/cv-linux-amd64.tar.gz")) | .url' "${WORK_DIR}/versions.json")
```

Comment:
A solution for using the param instead of hard-coding the version:
```suggestion
    # Using jq, parse list of versions to find download for the version required
    DOWNLOAD_URL_POSTFIX="${STATIC_LINTER_VERSION}/cv-linux-amd64.tar.gz"
    LINTER_TOOL_URL=$(jq -r --arg URL_POSTFIX "${DOWNLOAD_URL_POSTFIX}" '.[] | .assets | .[] |  select (.browser_download_url | endswith($URL_POSTFIX)) | .url' "${WORK_DIR}/versions.json")
```

Reply:
Changes have been applied